docs/msgs_package.md

```diff
@@ -22,7 +21,34 @@ For the encoder data stream the `edge_timestamp` is expected to be a constant ti
 
 <b>Note:</b>
 
-The LuciJoystickScaling message has the joystick values with the LUCI scaling applied. These are the values most recently used to control the chair and have already been sent to the wheelchairs drive controller. Please note this message also contains the scaling zone that was used for these outputs.
+The LuciJoystickScaling message has the joystick values with the LUCI scaling applied as well as the scaling applied. These are the values most recently used to control the chair and have already been sent to the wheelchairs drive controller. Please note this message also contains the scaling zone that was used for these outputs.
+
+Input source parameter is added to LuciJoystick and LuciJoystickScaling. This parameter denotes which input source has been set by the user. If this is used in any place to send commands to LUCI, make sure it is set to the same input as the desired/set state on LUCI or LUCI will disregard the command.
```

Comment:
I think an example could be helpful to illustrate this. Something like 

For example, when using the set_shared_remote_input service, the sent joystick must have the matching SharedRemote source set for those joystick values to be used..

Reply:
Ok that's fine